capytaine/io/mesh_loaders.py

```diff
@@ -739,7 +739,14 @@ def load_GDF(filename, name=None):
 
     ifile.close()
 
-    return Mesh(vertices, faces, name)
+    if isx == '1' and isy == '1':
+        return ReflectionSymmetricMesh(ReflectionSymmetricMesh(Mesh(vertices, faces, f"quarter_of_{name}"), yOz_Plane, f"half_of_{name}"), xOz_Plane, name)
+    elif isx == '1':
+        return ReflectionSymmetricMesh(Mesh(vertices, faces, f"half_of_{name}"), xOz_Plane, name)
```

Comment:
According to WAMIT's manual, `ISX = 1` means that the $x=0$ plane is a plane of symmetry. The $x=0$ plane would be `yOz_Plane` in Capytaine. Could you double check with an example?

Reply:
Yes, that is correct. Thanks for catching that. I only have an example that has both axes of symmetry, but the implementation at least functions for that example.